.github/actions/git-tag/action.yml

```diff
@@ -23,8 +23,15 @@ runs:
     - uses: actions/checkout@v3
       if: steps.checkout-test.outputs.checkout == 'true'
 
-    - name: Create tag
+    - name: Create tag it doesn't exist
       run: |
+        if git ls-remote --exit-code origin refs/tags/${{ inputs.tag }}; then
+          echo "Tag already exists, skipping"
+          exit 0
+        fi
+
+        echo "Tag doesn't exist, creating"
```

Comment:
```suggestion
        echo "::debug::Tag doesn't exist, creating"
```

Reply:
Not sure if this should be debug, having to rerun the workflow in a different mode is kind of the opposite of what we want for this?

Reply:
I just leave them as-is personally, it's good for logs you can check when something goes wrong but you don't need a whole annotation from GitHub (which is a good point for a skip warning above)